Sources/PexipScreenCapture/Shared/ScreenMediaCapturer.swift

```diff
@@ -43,6 +43,17 @@ public protocol ScreenMediaCapturerDelegate: AnyObject {
 public protocol ScreenMediaCapturer: AnyObject {
     var delegate: ScreenMediaCapturerDelegate? { get set }
 
+    #if os(iOS)
+
+    /**
+     Starts screen capture with the given fps.
+     - Parameters:
+        - fps: The FPS of a video stream (15...30)
+     */
+    func startCapture(atFps fps: UInt) async throws
```

Comment:
Correct me if I'm wrong, but this is a breaking change?

Reply:
Yes, do you think we should keep the old method and add deprecation warning? This protocol is public, but it's not something you would normally use unless you want to implement your custom screen capturer instead of the one we provide.

Reply:
Probably less important for this case, but as a rule of thumb we should deprecate for at least one release and only then remove the method.

Up to you how to best proceed here.

Reply:
Ok, added deprecation warning in https://github.com/pexip/pexip-swift-sdk/pull/18/commits/d38e2aca08438b9469e4e441ed50e6a2983fb5f0